### DISCUSSION TITLE
UGS Editor windoe

### DISCUSSION THREAD

**Discussion by bobgibson2908** [winder/Universal-G-Code-Sender#2537] [Help]
I am new to UGS and have it installed on my Macbook Pro(M2) and my iMac (intel).. On my iMac the gCode editor window will allow me to edit the gcode but on my Macbook I can only view and run the code.  Have I got a setting on my Macbook set incorrectly.

**Comment by breiler** [comment]
I looked through your logs that you sent by email and couldn't find any problems. You could try and delete the settings and see if that helps. As I only have an older iMac I doubt that I will be able to reproduce this problem, you should move on to another software: https://github.com/winder/Universal-G-Code-Sender/wiki/Software-comparison#gcode-sender

**Comment by bobgibson2908** [comment]
Thank you for your help.

How do I delete the settings. Are they in a file some here

Cheers

Bob

**Reply by breiler** [reply]
They should be located here (delete the `ugsplatform` directory):
`/Users/bobgibson/Library/Application Support/ugsplatform`

**Comment by bobgibson2908** [comment]
Hi

Yes that fixed it the editor window now has all the features.

Thank you appreciate your help

Cheers

Bob
